Resources/views/process_type/index.html.twig

```diff
@@ -21,6 +22,7 @@
                     <td><a href="{{ path('process_type_show', {'id': process_type.id}) }}">{{ process_type.name }}</a></td>
                     <td>{{ (process_type.hideInDashboard ? 'common.boolean.true' : 'common.boolean.false')|trans }}</td>
                     <td>{{ conclusionClassTranslation(process_type.conclusionClass) }}</td>
+                    <td>{{ process_type.netDefaultValue }}</td>
```

Comment:
Maybe we should use https://twig.symfony.com/doc/2.x/filters/number_format.html here? And in general when displaying numbers?

Reply:
Fixed in: https://github.com/aakb/kontrolgruppen-core-bundle/pull/34/commits/70dcb1fe6ccc2cada3c44731ad56aed46ce80de0